locales/en-US/editor.properties

```diff
@@ -193,6 +193,14 @@ AUDIO_OPTION_AUTOPLAY=Start playing the audio automatically
 AUDIO_OPTION_LOOPING=Restart the audio when it is over
 AUDIO_OPTION_DISABLE_AUDIO=Start audio player muted
 
+# Binary Viewer
+BINARY_FILE_TITLE=Non-editable File
+BINARY_FILE_CANNOT_EDIT=I'm not sure how to edit this file!
+BINARY_FILE_OPTIONS_TITLE=But you can...
```

Comment:
How is this displayed in the UI? Are the options below shown in a dropdown? If it is then it's fine. Otherwise if it is being attached to this string, you need a placeholder here for where the rest of the string will appear.